cpp/core/compute/Backend.h

```diff
@@ -118,7 +118,66 @@ class Backend : public std::enable_shared_from_this<Backend> {
   std::unordered_map<std::string, std::string> confMap_;
 };
 
-void setBackendFactory(std::function<std::shared_ptr<Backend>()> factory);
+using BackendFactory1 = std::shared_ptr<Backend> (*)(const std::unordered_map<std::string, std::string>&);
+using BackendFactory2 = std::shared_ptr<Backend> (*)();
```

Comment:
Can we rename `BackendFactory2` as `BackendFactory`?

Reply:
OK